### PR TITLE
DateBox should not raise any errors when useMaskBehavior is enabled and locale digits are different to arabic digits (T851630)

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: js/localization/ldml/date.parser.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 const escapeRegExp = require('../../core/utils/common').escapeRegExp;
+const checkDigit = require('./date.format').checkDigit;
+const Globalize = require('globalize');
```
Comment (DokaRus) [inline js/localization/ldml/date.parser.js]: We can't do that. Globalize is an optional dependency

**Review by DokaRus — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.localization/localization.globalize.widgets.tests.js [hunk 2]
```diff
@@ -86,6 +86,28 @@ QUnit.test('DateBox should localize whole date in arabic locale', function(asser
     }
 });
 
+QUnit.test('DateBox should not raise error when digits are not default arabic digits', function(assert) {
```
Comment (nightskylark) [inline testing/tests/DevExpress.localization/localization.globalize.widgets.tests.js:89]: Don't you think we should tests it in Intl too?

File: testing/runner/Views/Main/RunSuite.cshtml [hunk 3]
```diff
@@ -6,7 +6,8 @@
         || Model.ScriptVirtualPath.Contains("Bundles")
         || Model.ScriptVirtualPath.Contains("DevExpress.angular")
         || Model.ScriptVirtualPath.Contains("DevExpress.jquery");
-    var isIntlTest = Model.ScriptVirtualPath.Contains("intl.tests");
+    var isIntlTest = Model.ScriptVirtualPath.Contains("intl.tests")
```
Comment (San4es) [inline testing/runner/Views/Main/RunSuite.cshtml]: @nightskylark @DokaRus
Can we get rid of this and use `import` or `require` for Intl in corresponding test files?

File: testing/tests/DevExpress.localization/localization.globalize.widgets.tests.js (hunk 2 above)
Comment (DokaRus) [reply testing/tests/DevExpress.localization/localization.globalize.widgets.tests.js:89]: Done

**Review by DokaRus — DISMISSED** [review]

File: testing/runner/Views/Main/RunSuite.cshtml (hunk 3 above)
Comment (DokaRus) [reply testing/runner/Views/Main/RunSuite.cshtml]: Let's try it

**Review by DokaRus — APPROVED** [review]

**Review by San4es — APPROVED** [review]
LGTM

**Pull request merged**